mesonbuild/interpreter/interpreter.py

```diff
@@ -1662,23 +1662,28 @@ def func_build_target(self, node, args, kwargs):
         else:
             raise InterpreterException('Unknown target_type.')
 
-    @permittedKwargs({'input', 'output', 'fallback', 'command', 'replace_string'})
-    @FeatureDeprecatedKwargs('custom_target', '0.47.0', ['build_always'],
-                             'combine build_by_default and build_always_stale instead.')
```

Comment:
This could be removed as its own distinct thing. I suspect rebase errors, but in commit 06481666f4e74ecef01e59351fc345ab0962d998 this line got removed from build.py and added into the interpreter, but to the wrong function.

Since then, build_always became a typed kwarg that carries that info around natively, but it never had the warning. It should.

I've done a more specific fix in #9887